pipeline/backend/local/local.go

```diff
@@ -3,18 +3,19 @@ package local
 import (
 	"context"
 	"encoding/base64"
+	"github.com/woodpecker-ci/woodpecker/pipeline/backend/types"
```

Comment:
this will likely hit the linter

just use `gofumpt -w --extra .` to make sure ;)
-> in this just commit changes of the result and push - we will squashmerge pulls

Reply:
I used gofmt -w, but indeed the change was suspect.